.github/workflows/upgrade-liff-sdk.yml

```diff
@@ -0,0 +1,31 @@
+name: Upgrade LIFF SDK
+
+on:
+  schedule:
+    - cron: '0 10 * * 1'
+
+jobs:
+  update-dep:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - name: Check and Update dependencies
+        run: |
+          npx -p npm-check-updates ncu -u @line/liff
+          npm install
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
```

Comment:
We should set environment variables such as `GITHUB_TOKEN`, `GITHUB_WORKSPACE`. It seems not set yet.

Reply:
It seems the default setting is using `GITHUB_TOKEN`.

Reply:
You need to add in here https://github.com/line/liff-playground/settings/secrets/actions

Reply:
Is that automatically generated by Github?
https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret

Reply:
My mistake! Thank you for sharing it.